pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -182,6 +188,18 @@ type VaultAppRole struct {
 	SecretRef SecretKeySelector `json:"secretRef"`
 }
 
+type KubernetesAuth struct {
+	// The Kubernetes auth backend mount point if different from the default "kubernetes"
+	// +optional
+	Path string `json:"mountPath"`
+
+	// Secret containing a Kubernetes ServiceAccount JWT used for authenticating with Vault
+	SecretRef SecretKeySelector `json:"secretRef"`
+
+	// The Vault role to assume. A role binds a Kubernetes ServiceAccount with a set of Vault policies.
+	Role string `json:"role"`
```

Comment:
Do we have a default value here? If not, would be good to explicitly say it's required 😄